.github/workflows/lint-types.yaml

```diff
@@ -0,0 +1,40 @@
+name: Lint Generated Types
+on:
+  workflow_call:
+  push:
+    paths:
+      - 'packages/client/**'
```

Comment:
do you still need/want this to be triggered when only client changes are made? I think it may be sufficient to just trigger on server changes no?

Reply:
It's to capture the case of modifying the types file manually but it's probably better to just specify the exact path of that file instead of all client files.